lib/tasks/govuk_assets.rake

```diff
@@ -0,0 +1,12 @@
+require 'services'
+require 'asset_processor'
+
+namespace :govuk_assets do
+  desc 'Set uploaded state for all clean assets (uploaded/not_uploaded)'
+  task set_uploaded_state_for_all_clean_assets: :environment do
+    processor = AssetProcessor.new(scope: Asset.where(state: 'clean'))
```

Comment:
Should we use `Asset.unscoped.where(state: 'clean')` so that we also update the state of deleted assets?

Reply:
Yes. Well spotted! I'd completely forgotten about the deleted assets (again!).